nirc_ehr/resources/queries/dbo/q_vitals.sql

```diff
@@ -0,0 +1,19 @@
+SELECT anmEvt.ANIMAL_EVENT_ID as objectid,
```

Comment:
This looks like we'll only have one value per row (blood pressure, heart rate, etc) and the others will be null. Would it be useful to group based on Id, Date, etc to collapse multiple observations to a single row?

Reply:
yes makes sense, merging now and will ask Matthew on the next call.